### PR TITLE
 PRJ-373: Relay server support

### PR DESCRIPTION
**Pull request by IKrukov-HORIS**

### CODE REVIEW

File: projector-server-core/src/main/kotlin/org/jetbrains/projector/server/core/websocket/HttpWsServer.kt [hunk 1]
```diff
@@ -188,6 +267,7 @@ public abstract class HttpWsServer(host: InetAddress, port: Int) {
     }
 
     override fun onStart() {
+      logger.info { "Server started on host $host and port $port" }
```
Comment (SerVB) [inline projector-server-core/src/main/kotlin/org/jetbrains/projector/server/core/websocket/HttpWsServer.kt:270]: Can we add the same message to relay server?

File: projector-client-common/src/jsMain/kotlin/org/jetbrains/projector/client/common/misc/ParamsProvider.kt [hunk 2]
```diff
@@ -64,6 +64,8 @@ actual object ParamsProvider {
   actual val CLIPPING_BORDERS: Boolean
   val HOST: String
   val PORT: String
+  val SERVER_ID: String?
+  val ENABLE_RELAY: Boolean
```
Comment (SerVB) [inline projector-client-common/src/jsMain/kotlin/org/jetbrains/projector/client/common/misc/ParamsProvider.kt:68]: Please add a very brief description of these to parameters to the end of the list: https://github.com/JetBrains/projector-client/tree/master/projector-client-web#debugtest-parameters

**Review by SerVB — CHANGES_REQUESTED** [review]

File: projector-server-core/src/main/kotlin/org/jetbrains/projector/server/core/websocket/HttpWsServer.kt (hunk 1 above)
Comment (IKrukov-HORIS) [reply projector-server-core/src/main/kotlin/org/jetbrains/projector/server/core/websocket/HttpWsServer.kt:270]: Sure. I've added logger and connection message to HttpWsClient.

File: projector-client-common/src/jsMain/kotlin/org/jetbrains/projector/client/common/misc/ParamsProvider.kt (hunk 2 above)
Comment (IKrukov-HORIS) [reply projector-client-common/src/jsMain/kotlin/org/jetbrains/projector/client/common/misc/ParamsProvider.kt:68]: Done

File: projector-client-web/README.md [hunk 3]
```diff
@@ -66,6 +66,8 @@ Name | Type | Default value | Description
 `repaintInterval` | Int | `333` | Set interval of repainting that is needed to paint loaded images in ms.
 `cacheSize` | Int | `5M` | Set size of cache for images in Chars.
 `blockClosing` | Boolean | `true` | Enable blocking of accidental closing of the web page
+`enableRelay` | Presence | Not present | Establish connection to Projector server via relay. The address of relay server should be passed via `host` parameter. 
+`serverId` | String? | Not present | Identifier of Projector server to connect to, when relay is used.
```
Comment (SerVB) [inline projector-client-web/README.md:70]: Now it seems like we can merge these two parameters into single like `relayServerId: String?`

Comment (IKrukov-HORIS) [reply projector-client-web/README.md]: Agreed. Fixed.

**Pull request merged**